modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/checkpoint/CheckpointMarkersStorage.java

```diff
@@ -304,7 +304,7 @@ private WALPointer readPointer(File cpMarkerFile, ByteBuffer buf) throws IgniteC
 
             return new FileWALPointer(buf.getLong(), buf.getInt(), buf.getInt());
         }
-        catch (IOException e) {
+        catch (Exception e) {
```

Comment:
What about writing a test for the issue?

Reply:
Done.